uniffi_bindgen/src/bindings/kotlin/gen_kotlin/miscellany.rs

```diff
@@ -0,0 +1,164 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use crate::bindings::backend::{CodeOracle, CodeType, Literal};
+use askama::Template;
+use paste::paste;
+use std::fmt;
+
+#[allow(unused_imports)]
+use super::filters;
+
+macro_rules! impl_code_type_for_miscellany {
```

Comment:
This PR takes no view on whether or not this is a sensible resting place for these types. If it is, then this would also be a good place to put JSON and GUID.

If it isn't, then this would be a good place to put helpers to make adapter/transform `CodeType`s.